cmd/romie.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+var debugFlag bool
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "romie",
+	Short: "Game ROM Manager",
+	Long: `romie is a command line interface for managing game ROMs.
+
+	Find more information at: https://romie-gr.github.io/romie/`,
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.romie.yaml)")
+	rootCmd.PersistentFlags().BoolVar(&debugFlag, "debug", false, "set verbose logs (developers)")
```

Comment:
```suggestion
	rootCmd.PersistentFlags().BoolVar(&debugFlag, "debug", false, "set verbose logs")
```

Don't agree with the `(developers)` part.
For debugging, like when supporting a bug or question, it's also pretty useful.